integration-testing/rnode_testing/casper_propose_and_deploy.py

```diff
@@ -27,15 +30,15 @@ def deploy_block(i, node, expected_string, contract_name):
     logging.debug(f"Propose result: {exit_code}, output: {output}")
 
 
-def check_blocks(i, node, expected_string):
+def check_blocks(blocks_num, node, expected_string, network, config):
     logging.info(f"Check all peer logs for blocks containing {expected_string}")
 
     other_nodes = [n
                     for n in network.nodes
                     if n.container.name != node.container.name]
 
     for node in other_nodes:
-        wait_for(string_contains(show_blocks(node), expected_string),
+        wait_for(string_contains(show_blocks(node, blocks_num + 1), expected_string),  # i plus 1 genesis block
```

Comment:
This is prone to off-by-one errors because in these situations you never know if you should start at 0 or at 1 and when to correct the index with a `+1`... 

How can we avoid the `+1`?

Reply:
I don't really find a good way for that.